src/main/kotlin/no/nav/familie/dokument/storage/google/GcpStorage.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.familie.dokument.storage.google
+
+import com.google.cloud.storage.BlobId
+import com.google.cloud.storage.BlobInfo
+import com.google.cloud.storage.Storage
+import no.nav.familie.dokument.GcpDocumentNotFound
+import org.apache.commons.io.IOUtils
+import org.slf4j.LoggerFactory
+import java.io.InputStream
+
+class GcpStorage(private val bucketName: String, maxFileSizeMB: Int, private val storage: Storage) {
+
+    private val maxFileSizeAfterEncryption: Int = (maxFileSizeMB.toDouble() * 1000.0 * 1000.0 * ENCRYPTION_SIZE_FACTOR).toInt()
```

Comment:
I don't think this is so good.
1. should be * 1024 
2. When converting to pdf, the pdf is much larger than the image, in my test the image was 4.5mb and the pdf was 7.1mb